businesscentral/image.yaml

```diff
@@ -0,0 +1,26 @@
+name: "jboss-bpmsuite-7/bpmsuite70-businesscentral"
+version: "7.0.0"
+from: "jboss-eap-7/eap70:latest"
+user: 185
+envs:
+    information:
+        - name: "JBOSS_PRODUCT"
+          value: "bpmsuite"
+        - name: "JBOSS_BPMSUITE_VERSION"
+          value: "7.0.0.CAP01"
```

Comment:
The problem is that originally, I don't know what it will be called, as it is going to initially be a limited availability release.